ftw/solr/handlers.py

```diff
@@ -0,0 +1,190 @@
+from Products.CMFCore.utils import getToolByName
+from ftw.solr.converters import CONVERTERS
+from ftw.solr.interfaces import ISolrIndexHandler
+from logging import getLogger
+from plone.indexer.interfaces import IIndexableObject
+from plone.namedfile.interfaces import INamedBlobFile
+from plone.rfc822.interfaces import IPrimaryFieldInfo
+from zope.component import queryMultiAdapter
+from zope.interface import implementer
+import logging
+
+
+logger = getLogger('ftw.solr.handlers')
+logger.setLevel(logging.DEBUG)
```

Comment:
so this will log in debug level by default? is this necessary?

Reply:
That was just for debugging purpose. Removed.